index.es6.js

```diff
@@ -73,10 +80,10 @@ export default React.createClass({
       <div className={classes} onClick={this.handleClick}>
         <div className="react-toggle-track">
           <div className="react-toggle-track-check">
-            <Check />
+            {this.props.showIcons ? <Check /> : ''}
           </div>
           <div className="react-toggle-track-x">
-            <X />
+            {this.props.showIcons ? <X /> : ''}
```

Comment:
Could you use `null` intead of empty string literals for these? IMO, it makes it clearer what we're trying to do. :)